query-service-impl/src/test/resources/application.conf

```diff
@@ -39,6 +39,18 @@ service.config = {
         startTimeAttributeName = "Trace.start_time_millis"
         slowQueryThresholdMs = 0
         percentileAggFunction = "PERCENTILE"
+        additionalTenantFilters = [
+          {
+            "tenantId" = "__default"
+            "timeRangeAndFilters" = [
+              {
+                "startTimeMillis" = 1
+                "endTimeMillis" = 2
+                "filter" : "{\r\n  \"lhs\": {\r\n    \"columnIdentifier\": {\r\n      \"columnName\": \"SERVICE.name\"\r\n    }\r\n  },\r\n  \"operator\": \"EQ\",\r\n  \"rhs\": {\r\n    \"literal\": {\r\n      \"value\": {\r\n        \"string\": \"dummyService\"\r\n      }\r\n    }\r\n  }\r\n}"
```

Comment:
Q: Do we need `\r\n`? If we just put a valid single line json string, would it work?

Reply:
Yeah, it would work. Was just added by the json string escaper I was using, got rid of it.